pytest/tests/sanity/gc_sync_after_sync.py

```diff
@@ -51,6 +51,8 @@
 
 logger.info('Starting back node 1')
 nodes[1].start(boot_node=nodes[1])
+# State Sync makes the storage seem inconsistent.
+nodes[1].stop_checking_store()
```

Comment:
Shouldn't you do it before the node is restarted? Just to rule out any race conditions?

Reply:
`check_store` is a python test flag used below in `wait_for_blocks`, so by the look of it the execution order should be well defined